hs-bindgen/src-internal/HsBindgen/Frontend/Pass/Parse/Decl/Monad.hs

```diff
@@ -190,6 +180,12 @@ recordNonSelectedDecl curr = do
         -- use descriptive binding specification with anonymous declarations, we
         -- __must__ select these declarations.
         DeclAnon{} -> return ()
+      -- We intentionally do selection as part of parsing, rather than a
```

Comment:
I moved the note here, since I extracted the conversion function from `CXCursorKind` to `NameKind`.